cms/wizards/wizard_pool.py

```diff
@@ -14,82 +15,63 @@ def entry_choices(user, page):
     Yields a list of wizard entries that the current user can use based on their
     permission to add instances of the underlying model objects.
     """
-    for entry in wizard_pool.get_entries():
+    for entry in get_entries():
         if entry.user_has_add_permission(user, page=page):
             yield (entry.id, entry.title)
 
 
 class WizardPool(object):
-    _entries = {}
-    _discovered = False
-
-    def __init__(self):
-        self._reset()
-
-    # PRIVATE METHODS -----------------
-
-    def _discover(self):
```

Comment:
After thinking about this later, I'm getting the impression I may have messed up backwards compatibility with this. Because although the method `wizard_pool.register` still works, there is now no code that actually looks for cms_wizards.py
@czpython Where would you like the code that searches for cms_wizards.py files to live now?

Reply:
Ok, this is bizarre. Just wrote a test with an app that doesn't have cms_config.py but only cms_wizards.py and it seems to have registered the wizard from cms_wizards.py without me adding this into the init! Will check what's up there Monday morning, but possibly all that is needed is that test?

Reply:
Likely there is an import somewhere of cms_wizards.py which should be removed

Reply:
So, I can't use the `__init__` in CMSCoreExtensions due to the order in which things happen (it means attempting to add wizards to a CMSCoreExtensions instance that doesn't exist yet). However, I have added this elsewhere. Hope this will work for you.